cpp/src/io/parquet/page_data.cu

```diff
@@ -60,7 +60,7 @@ struct page_state_s {
     int32_t nz_count;               // number of valid entries in nz_idx (write position in circular buffer)
     int32_t dict_pos;               // write position of dictionary indices
     int32_t out_pos;                // read position of final output
-    int32_t int64_nanoscale;        // if non-zero, convert timestamp to nano
+    int32_t ts_scale;               // if non-zero, convert timestamp to nano
```

Comment:
Just to double check, Is the comment "// if non-zero, convert timestamp to nano" still valid?

Reply:
Nope -> updated the comment.